config/locales/ar.yml

```diff
@@ -2653,7 +2653,6 @@ ar:
   spree_admin_single_enterprise_hint: "لمحة: للسماح للأشخاص بالعثور عليك ، قم بتشغيل الرؤية الخاصة بك أسفل"
   spree_admin_eg_pickup_from_school: "على سبيل المثال. &quot;البيك اب من المدرسة الابتدائية&quot;"
   spree_admin_eg_collect_your_order: "على سبيل المثال. &quot;يرجى جمع طلبك من 123 Imaginary St، Northcote ، 3070&quot;"
-  spree_classification_primary_taxon_error: "Taxon %{taxon} هو التصنيف الأساسي %{product} ولا يمكن حذفه"
```

Comment:
We never change other locales than en.yml because they are updated automatically through our Transifex integration.

Reply:
> We never change other locales than en.yml because they are updated automatically through our Transifex integration.

OK.I will update it.